### PR TITLE
Feature/update create a skin page

### PR DESCRIPTION
**Pull request by lostkeys**
This fixes the docs for making a skin being outdated and wrong with the new monorepo. 

Has to be coordinated with TINY-3803 in the Monorepo.

### CODE REVIEW

File: advanced/creating-a-skin.md [hunk 1]
```diff
@@ -7,78 +7,80 @@ description: Introducing skin creation, less and icon modification.
...
+* Basic understanding of [Less](http://lesscss.org), the CSS preprocessor we use to build the skins. More specifically, [read the section about variables](http://lesscss.org/features/#variables-feature).
 
 ## Preparation
 
-A skin for TinyMCE 5 is written in [less](http://lesscss.org), a popular CSS preprocessor. Making a skin primarily involves modifying variables similar to using an API. The CSS rules are not modified or overridden, instead the `less` variables in the `skin` files is modified. This makes updating TinyMCE easier.
+The CSS that goes with a theme is called a skin. The default skin for TinyMCE 5 is named Oxide and is written in [Less](http://lesscss.org), a popular CSS preprocessor. With Oxide we introduced a concept we call the **Style API**. This API consists of around 300 variables which you use to modify the appearance of TinyMCE. You never touch the underlying CSS. The benefit of this approach is that improvements we make to the the CSS and HTML won't break your custom skin. This also means that if things doesn't work as expected, we can provide support and bug fixes, something that was virtually impossible before.
```
Comment (lnewson) [inline advanced/creating-a-skin.md]: `if things doesn't work as expected` -> `if things don't work as expected` maybe?

File: advanced/creating-a-skin.md [hunk 2]
```diff
@@ -7,78 +7,80 @@ description: Introducing skin creation, less and icon modification.
...
 
-The preparation steps above must be performed to create or edit a skin.
+Make sure you have performed the preparation steps above.
 
 ### Overview
 
-In the `/oxide` directory, navigate to `src/less/skins/`. There are two folders in this location:
-* `/ui` - which is the skins for the editor.
+Navigate to `modules/oxide/src/less/skins/`. There are two folders in this location:
+* `/ui` - which is the skins for the editor. The important file here is  `skin.less`.
```
Comment (lnewson) [inline advanced/creating-a-skin.md]: Minor typo here, as there's a double space.

File: advanced/creating-a-skin.md [hunk 3]
```diff
@@ -7,78 +7,80 @@ description: Introducing skin creation, less and icon modification.
...
 * `/content` - which is the skins for the content within the editor.
 
-The skin imports the theme `less` files located in the `src/less/theme/` directory.
+The folder `modules/oxide/src/less/theme/` contains the less files. At the top of most files you find the available variables that defines the default colors, margins, fonts etc (variables are the strings that starts with an at-character, for example `@background-color`). *Do not edit these files*, instead use them as a reference when creating your skin. Two common files you likely will start with are the global variables: `modules/oxide/src/less/theme/globals/global-variables.less` and the toolbar buttons: `modules/oxide/src/less/theme/components/toolbar-button/toolbar-button.less`.
```
Comment (lnewson) [inline advanced/creating-a-skin.md]: `At the top of most files you find the available variables` -> `At the top of most files you'll find the available variables` maybe?

File: advanced/creating-a-skin.md [hunk 4]
```diff
@@ -88,76 +90,33 @@ Following is an example of a sample `skin.less` file:
...
+> Tip: You can change the TinyMCE config in `modules/oxide/src/demo/index.html` to suit your particular use case.
 
-TinyMCE editor with all the configured skin changes will appear in the browser screen.
+To style interface elements in the content, such as selection color, drag handles, table of contents, bookmarks etc, copy variables to `modules/oxide/src/less/skins/ui/<skin-name>/content.less` or `content.inline.less` (if inline mode is being used).
 
-### Creating a content CSS file
+## Creating a content CSS file
 
-To update the appearance of the content within the editor, such as headings, quotes, lists etc you create a content css. These are located in `src/less/skin/content/`
+To update the appearance of the content within the editor, such as headings, quotes, lists etc you create a content css. These are located in `modules/oxide/src/less/skin/content/`
```
Comment (lnewson) [inline advanced/creating-a-skin.md]: `lists etc` -> `lists, etc...` maybe?

File: advanced/creating-a-skin.md (hunk 3 above)
Comment (blackcathikari) [reply advanced/creating-a-skin.md]: `less` should probably also be capitalised, in the first sentence of this paragraph. I read it as the "less than" definition at first, and got a bit confused. 

`Two common files you likely will start with are the global variables:` is a bit awkward to read too. Might be better as something like `We recommend starting with the two files containing global variables:`?

File: advanced/creating-a-skin.md [hunk 5]
```diff
@@ -88,76 +90,33 @@ Following is an example of a sample `skin.less` file:
...
+This is how you skin TinyMCE: copy variables from the files in the theme folder and paste them into your skin file. There are variables for most things, like spacing between toolbar buttons to letter spacing. Simple yet powerful.
 
-#### Result
+> Tip: You can change the TinyMCE config in `modules/oxide/src/demo/index.html` to suit your particular use case.
 
-TinyMCE editor with all the configured skin changes will appear in the browser screen.
+To style interface elements in the content, such as selection color, drag handles, table of contents, bookmarks etc, copy variables to `modules/oxide/src/less/skins/ui/<skin-name>/content.less` or `content.inline.less` (if inline mode is being used).
 
-### Creating a content CSS file
+## Creating a content CSS file
```
Comment (blackcathikari) [inline advanced/creating-a-skin.md]: This section has the same information as the paragraph above, on line 107? Could we combine line 107 into this section?

File: advanced/creating-a-skin.md [hunk 6]
```diff
@@ -7,78 +7,80 @@ description: Introducing skin creation, less and icon modification.
...
+
+## Prerequisites
 
 This guide assumes:
 
-* Basic understanding of [Node.js](http://nodejs.org) and [Gulp](http://gulpjs.com).
 * Familiarity with the command line and running simple commands.
-
-> Note: The  [TinyMCE Skin Creator](http://skin.tinymce.com/) only supports TinyMCE 4.
+* Having [NodeJS](https://nodejs.org/en/), [Yarn](https://yarnpkg.com/en/) and [Gulp](https://gulpjs.com) installed
```
[the remark this answers is not in the document]
Comment (lostkeys) [reply advanced/creating-a-skin.md]: I wrote it like this
> NodeJS, Yarn and Gulp are already installed.

File: advanced/creating-a-skin.md (hunk 5 above)
Comment (lostkeys) [reply advanced/creating-a-skin.md]: I removed that, as I believe it was left there by mistake.

**Pull request merged**